src/components/PostComment.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+import { postComment } from '../../utils/api';
+import { username } from '../App';
+
+export default function PostComment({ article_id, setComments }) {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(null);
+  const [comment, setComment] = useState('');
+
+  const onChangeHandler = (e) => {
+    setComment(e.target.value);
+  };
+
+  const saveComment = (e) => {
+    e.preventDefault();
+    setIsLoading(true);
+    postComment(article_id, {
+      username: username,
+      body: comment,
+    })
+      .then((response) => {
+        if (response.status === 201) {
+          alert('Posted');
```

Comment:
I'd recommend trying to stay away from `alert` as it's a bit of an old fashioned approach these days and might get blocked by a user's popup-blocking extension if they have one

Reply:
yeah, using it just for debug - i can see preloader while it is stays opened =))